### DISCUSSION TITLE
Make the library available on the documentation page

### DISCUSSION THREAD

**Discussion by mpartel** [js-fns/date-fns#2851] [Q&A] [answered]
When browsing the docs, it'd be convenient to be able to open the developer console and have the functions available in global scope.

**Comment by fturmel** [comment, marked as answer]
Hi @mpartel,

Very late reply, but I think what you were asking for was either already available or has been implemented since.

If you look at your network tab, you'll see that when invoking `init()` it pulls the latest version of the library from the [unpkg CDN](https://unpkg.com/date-fns/esm/index.js) and makes it available through the `_` global variable.

<img width="1365" alt="Screen Shot 2021-12-15 at 10 07 48 AM" src="https://user-images.githubusercontent.com/4009209/146211561-32042804-bf5b-4022-9171-0fb06539aa28.png">

**Reply by gabrielbergoc** [reply]
Is it possible to make the FP package available too?
